.github/workflows/tests.yml

```diff
@@ -11,6 +11,13 @@ jobs:
         django-version:
           - "3.1"
           - "3.2"
+          - "4.0"
+          - "4.1"
+        exclude:
+          - python-version: "3.6"
```

Comment:
It looks like 3.7 isn't supported with django 4.x either. I'd suggest just changing the matrix versions above to 3.8 and 3.9 and just dropping the `exclude` block altogether. keeps it simple